OrbitQt/GgpInstanceItemModel.h

```diff
@@ -0,0 +1,36 @@
+#ifndef ORBITQT_GGP_INSTANCE_ITEM_MODEL_H_
+#define ORBITQT_GGP_INSTANCE_ITEM_MODEL_H_
+
+#include <qobjectdefs.h>
```

Comment:
Nit: That looks like an internal Qt Header. Probably we should include <QObject> here instead.

Reply:
done